lib/languages/dart.dart

```diff
@@ -4,83 +4,151 @@ import 'package:highlight/highlight_core.dart';
 import '../LanguagesModes/common_modes.dart';
```

Comment:
Added redundant types.
Formatting is a bit better but still suffers without trailing commas.
Will require more work, no benefit of merging this.